.pre-commit-config.yaml

```diff
@@ -12,10 +12,9 @@ repos:
     rev: v6.13.1
     hooks:
     -   id: ansible-lint
-        args: ["-c", ".config/ansible-lint.yml"]
```

Comment:
Why did this file need to move?

Reply:
The ansible-linter had trouble finding it in the .config location. I moved it and it immediately detected it and worked.

Reply:
interesting ... do you know what the problem might've been? I'm ok at the top level, but I'd be curious to know what was up with the linter path discovery itself

Reply:
I didn't bother checking implementation, but I did see an issue where someone mentioned it. That prompted my changes. https://github.com/ansible/ansible-lint-action/issues/155